src/pages/core-candy-machine/guides/create-a-core-candy-machine-ui.md

```diff
@@ -0,0 +1,789 @@
+
+Assuming again a Candy Machine with `startDate`, `SolPayment` and `mintLimit` Guards attached let's see how to build the `mintArgs`.
+
+```ts
+import { some, unwrapOption } from '@metaplex-foundation/umi';
+import {
+  DefaultGuardSetMintArgs
+} from "@metaplex-foundation/mpl-core-candy-machine";
+
+let mintArgs: Partial<DefaultGuardSetMintArgs> = {};
+
+// add solPayment mintArgs
+const solPayment = unwrapOption(candyGuard.guards.solPayment)
+if (solPayment) {
+  mintArgs.solPayment = some({
+    destination: solPayment.destination,
+  });
+}
+
+// add mintLimit mintArgs
+const mintLimit = unwrapOption(candyGuard.guards.mintLimit)
+if (mintLimit) {
+  mintArgs.mintLimit = some({ id: mintLimit.id });
+}
+```
+
+Not all Guards require additional `mintArgs` to be passed in. This is the reason `startDate` is not in the above code snippet. To understand if the guards you are using require `mintArgs` to be passed in it is recommended to check the [Developer Hub](/core-candy-machine) Guard pages. If there are "Mint Settings" described you need to pass in `mintArgs` for this guard.
+
+Now that the `mintArgs` are built let's see how to call the mint function itself. The following snippet assumes that the `candyMachine` and `candyGuard` were fetched as described above. Technically the publicKeys of `candyMachine`, `collection`, `candyGuard` and all the `mintArgs` can also be passed in manually in case you do not want to fetch them.
+
+```ts
+// Generate the NFT address
+const nftMint = generateSigner(umi);
+
+await mintV1(umi, {
+  candyMachine: candyMachine.publicKey,
+  collection: candyMachine.collectionMint,
+  asset: nftMint,
+  candyGuard: candyGuard.publicKey,
+  mintArgs,
+}).sendAndConfirm(umi)
+
+console.log(`NFT ${nftMint.publicKey} minted!`)
+```
+
+
+## Advanced Minting Techniques
+
+While the basic minting function we've discussed works well for most cases, there are some advanced techniques you can use to enhance your minting process. Let's explore a couple of these:
+
+### Minting Multiple NFTs in One Transaction
+
+For efficiency, you might want to allow users to mint multiple NFTs in a single transaction. Here's how you can achieve this:
+
+Depending on the specific setup it can be helpful to allow minting multiple NFTs in one Transaction by combining the [Transaction Builders](/umi/transactions#transaction-builders).
+
+```ts
+let builder = transactionBuilder()
+  .add(mintV1(...))
+  .add(mintV1(...))
+```
+
+In case you allow to mint many Assets in one transaction you might experience `Transaction too large` errors. The function [`builder.fitsInOneTransaction(umi)`](/umi/transactions#transaction-builders) allows to check for this before sending the transaction so that the transaction can be split before sending it. In case splitting is needed using [`signAllTransactions`](/umi/transactions#building-and-signing-transactions) is recommended so that only one popup has to be approved in the Wallet Adapter.    
```

Comment:
```suggestion
If you add to many `mintV1` instructions into a transaction you will recieve a `Transaction too large` error. The function [`builder.fitsInOneTransaction(umi)`](/umi/transactions#transaction-builders) allows to check for this before sending the transaction so that the transaction can be split before being sentt. In case splitting is needed using [`signAllTransactions`](/umi/transactions#building-and-signing-transactions) is recommended so that only one popup has to be approved in the Wallet Adapter.    
```